combustion

```diff
@@ -100,15 +101,17 @@ if [ "${1-}" = "--prepare" ]; then
 			# Set rd.neednet if not already done and reevaluate it (module-specific)
 			getargbool 0 'rd.neednet' && exit 0
 			echo rd.neednet=1 > /etc/cmdline.d/40-combustion-neednet.conf
-			if [ -e /lib/dracut/hooks/pre-udev/60-net-genrules.sh ]; then
+			if [ -e "\$hookdir"/pre-udev/60-net-genrules.sh ]; then
 				# Wicked
-				. /lib/dracut/hooks/pre-udev/60-net-genrules.sh
+				. "\$hookdir"/pre-udev/60-net-genrules.sh
 				# Re-trigger generation of network rules and apply them
 				udevadm control --reload
 				udevadm trigger --subsystem-match net --action add
-			else
+			elif [ -e "\$hookdir"/cmdline/99-nm-config.sh ]; then
 				# NetworkManager
-				. /lib/dracut/hooks/cmdline/99-nm-config.sh
+				. "\$hookdir"/cmdline/99-nm-config.sh
+			else
+				echo "ERROR: unknown network framework" >&2
```

Comment:
If it's an error it should `exit 1`

Reply:
was hoping to get away with incomplete networkd support for now